app/views/registrations/new.html.erb

```diff
@@ -0,0 +1,40 @@
+<div class="vh-100 d-flex justify-content-center align-items-start">
+  <div class="w-50 mt-5">
+    <h2 class="text-center">Sign up</h2>
+
+    <%= form_for(resource, as: resource_name, url: registration_path(resource_name), html: { class: "mt-4" }) do |f| %>
+      <%= render "devise/shared/error_messages", resource: resource %>
+
+      <div class="form-group">
+      <%= f.label :name %>
+      <%= f.text_field :name, autofocus: true, autocomplete: "name", class: "form-control" %>
+      </div>    

```

Comment:
- [ ] @Team Great work adding a page to register new users, I am having an issue, the name field doesn't get displayed as shown below, kindly ensure all your pages are working correctly after fixing the error below:

![image](https://github.com/EvansSnave/Blog_app_rails/assets/61408860/f75961ee-13fe-4386-817a-3b49c4da0a03)

Reply:
- [ ] @Team, Great work on your login and registration page, however there design doesn't match the wireframe design given below is a comparison with the registration page, the text field and buttons should be of the same width:


Wireframe Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://github.com/EvansSnave/Blog_app_rails/assets/61408860/7a69cd41-4517-43d6-89f9-d626efe846a3)  | ![image](https://github.com/EvansSnave/Blog_app_rails/assets/61408860/9f65cf3f-3974-44af-abe2-b49b10e52dde)